frontend/src/components/commons/Translation.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import LOCALIZE from "../../text_resources";
+
+const LANGUAGES = {
+  english: "en",
+  french: "fr"
+};
+
+class Translation extends Component {
+  state = {
+    curLanguage: LANGUAGES.english
+  };
+
+  static propTypes = {
+    updateLanguageOnPage: PropTypes.func
+  };
+
+  onSetLanguageToFrench = async () => {
```

Comment:
why are these functions `async`? why not just `= () => {`?

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/63/commits/c00c266ebbb51dff91acfeed1bcc3a3095b7b27b

I put that there when I was trying to fix my issues with the component refresh. So good catch, we don't need it.